Lib/test/datetimetester.py

```diff
@@ -2793,6 +2794,7 @@ def test_strptime_single_digit(self):
                 newdate = strptime(string, format)
                 self.assertEqual(newdate, target, msg=reason)
 
+    @warnings_helper.ignore_warnings(category=DeprecationWarning)  # gh-70647
```

Comment:
I think we can omit this comment since just a two lines below we have the same explanation :)

Reply:
```suggestion
    @warnings_helper.ignore_warnings(category=DeprecationWarning)
```